bpf/lib/process.h

```diff
@@ -313,8 +314,15 @@ struct binary {
 	// but can contain negative value in case of copy error.
 	// While s16 would be sufficient, 64 bits are handy for alignment.
 	__s64 path_length;
+	// if end_r contains reversed path postfix
+	__u8 reversed;
+	__u8 pad[7];
```

Comment:
If you want you can also make `path_length` a u32 and reverse a u32 so that we are still aligned.